src/test/java/com/google/api/codegen/testdata/discoveries/csharp/csharp_appengine.v1beta5.json.baseline

```diff
@@ -17,6 +17,7 @@
 using Google.Apis.Appengine.v1beta5;
 using Google.Apis.Auth.OAuth2;
 using Google.Apis.Services;
+using Newtonsoft.Json;
```

Comment:
Newtonsoft.Json is included transitively, so it's not necessary to explicitly add it as a dep to the project.
I.e. Google.Apis.* depends on Google.Apis, which depends on Google.Apis.Core, which depends on Newtonsoft.Json